lua/telescope/builtin/__files.lua

```diff
@@ -384,6 +384,7 @@ files.find_files = function(opts)
   pickers
     .new(opts, {
       prompt_title = "Find Files",
+      files_picker = true,
```

Comment:
lets rename this to `__cursor_support` (i think, important thing is the `__` prefix)

Reply:
or `__location_input` dont care.